tests/dummy/app/router.js

```diff
@@ -16,6 +16,11 @@ Router.map(function() {
     this.route('message', {path: "/:id"});
   });
 
+  // Routes to test focusing outlet on last route in a route path
+  this.route('about', function() {
+    this.route('us');
```

Comment:
We can probably test this at just the `about` level, no need for the `us` route? If it doesn't fail with just `about` I'd like to add test cases for both. :smile: